client-tools/Import/ImportFeature.cpp

```diff
@@ -165,7 +165,7 @@ void ImportFeature::collectOptions(
 
   options->addOption("--remove-attribute",
                      "remove an attribute before inserting documents"
-                     " into collection (for CSV and TSV only)",
+                     " into collection (for CSV, TSV and JSON only)",
```

Comment:
JSON, JSONL, or both? If both, what format is left?

Reply:
JSON and JSONL,
left is XML

Reply:
We don't support importing XML. It's only available in arangoexport.